File: backend/tour-design-api/src/main/java/com/tourdesign/platform/model/PackageRestaurantModel.java
package com.tourdesign.platform.model;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PackageRestaurantModel {
    private Long id;
    private Integer orderIndex;

    private TravelPackageModel travelPackage;
    private RestaurantModel restaurant;
}

File: frontend/tour-design-web/src/app/models/package-restaurant.model.ts
// src/app/models/package-restaurant.model.ts
import { TravelPackageModel } from './travel-package.model';
import { RestaurantModel } from './restaurant.model';

export interface PackageRestaurantModel {
  id: number;
  orderIndex?: number;
  travelPackage?: TravelPackageModel;
  restaurant?: RestaurantModel;
}
